### DISCUSSION TITLE
How to disable releases along with "Now you can release this change" and "This PR seems to be unreleased" after PR merges?

### DISCUSSION THREAD

**Discussion by archon810** [community/community#20894] [Pull Requests]
We have a private repo that has never used Github releases, but today I noticed this after merging, which seems new (and there are no relevant Google hits either):

> Now you can release this change

and if I refresh the PR page, it changes to:

> This PR seems to be unreleased

I don't want to use releases at all, how can they be disabled, along with these messages?

![image](https://user-images.githubusercontent.com/469543/179832626-13dad555-20c7-4c8e-9659-1a05822b88be.png)

**Comment by baysideengineer** [comment]
Hey @archon810, I think this is a new feature where GitHub is trying to raise your attention for features you may have not used yet. I also received something similar about my branches and that I should “add some protection rules”. I didn’t found a checkbox in settings so I think this will stay and hopefully disappears just as quietly as it came. I hope this helps you for now.

**Reply by archon810** [reply]
Right, but such features should have an off setting - you can turn off wikis, projects, discussion, etc. but why not releases and the relevant UI notifications?

**Reply by baysideengineer** [reply]
I am totally with you. Let's hope that GitHub will implement a feature to turn such notification on/off. If you like you can [open a new poll](https://github.com/orgs/github-community/discussions/new) and let users vote for your idea. My vote is safe!
